<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="PC-201310021849" name="ranxiangxue.selenium.maven1.runcase.LoginCase" tests="1" failures="0" timestamp="24 一月 2021 15:30:02 GMT" time="3.218" errors="1">
  <testcase name="case1" time="3.218" classname="ranxiangxue.selenium.maven1.runcase.LoginCase">
    <error type="java.lang.IllegalArgumentException" message="Must provide a location for a move action.">
      <![CDATA[java.lang.IllegalArgumentException: Must provide a location for a move action.
	at org.openqa.selenium.interactions.MoveMouseAction.<init>(MoveMouseAction.java:34)
	at org.openqa.selenium.interactions.Actions.moveToElement(Actions.java:257)
	at ranxiangxue.selenium.maven1.page.BasePage.MoveToElement(BasePage.java:53)
	at ranxiangxue.selenium.maven1.handle.LoginHandle.GetUserText(LoginHandle.java:39)
	at ranxiangxue.selenium.maven1.runcase.LoginCase.case1(LoginCase.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- case1 -->
</testsuite> <!-- ranxiangxue.selenium.maven1.runcase.LoginCase -->
